seahorn/jobs/string_new_from_string/aws_string_new_from_string_harness.c

```diff
@@ -12,7 +12,7 @@
 
 int main() {
     /* parameters */
-    struct aws_string *source = ensure_string_is_allocated_bounded_length(SIZE_MAX - 1 - sizeof(struct aws_string));
+    struct aws_string *source = ensure_string_is_allocated_nondet_length();
```

Comment:
call `ensure_string_is_allocated_nondet_length()` is equilvance to call `ensure_string_is_allocated_bounded_length(SIZE_MAX - 1 - sizeof(struct aws_string))` for seahorn.
The definition is here:
https://github.com/seahorn/verify-c-common/blob/6bd27abc24af7140e2e2270afd8ce03fef1a9de0/seahorn/lib/string_helper.c#L29-L34